war/src/main/less/base/layout-commons.less

```diff
@@ -80,7 +80,7 @@ body {
 .app-page-body--one-column {
   --form-item-max-width: calc(85vw - 4rem);
 
-  @media screen and (max-width: 768px) {
+  @media screen and (max-width: 970px) {
```

Comment:
I can't push but changing this to 1200px fixed the behaviour Daniel posted about

Reply:
```suggestion
  @media screen and (max-width: 1200px) {
```